web-rdfs/source/http_handlers.cc

```diff
@@ -77,25 +99,30 @@ void get_handler(std::shared_ptr<HttpsServer::Response> response,
     delete_file_handler(response, path);
   } else if (!typeOfRequest.compare("OPEN")) {
     read_file_handler(response, path);
-  } else {
-    create_file_handler(response, request);
+  } else if (!typeOfRequest.compare("CREATE")) {
+    create_file_handler(response, path);
+  } else if (!typeOfRequest.compare("LISTSTATUS")) {
+    ls_handler(response, request, path);
   }
 }
 
+void not_found_handler(std::shared_ptr<HttpsServer::Response> response,
```

Comment:
I don't think this is necessary anymore; there was a patch by @vrnaz to handle this in #201.